Source/Fluids/WarpXFluidContainer.H

```diff
@@ -0,0 +1,172 @@
+
+    void InitData (int lev, amrex::Box init_box, amrex::Real cur_time);
+
+    void ReadParameters ();
+    /**
+     * Evolve updates a single timestep (dt) of the cold relativistic fluid eqautions
+     */
+    void Evolve (int lev,
+        const amrex::MultiFab& Ex, const amrex::MultiFab& Ey, const amrex::MultiFab& Ez,
+        const amrex::MultiFab& Bx, const amrex::MultiFab& By, const amrex::MultiFab& Bz,
+        amrex::MultiFab* rho, amrex::MultiFab& jx, amrex::MultiFab& jy, amrex::MultiFab& jz,
+        amrex::Real cur_time, bool skip_deposition=false);
+
+    /**
+     * AdvectivePush_Muscl takes a single timestep (dt) of the cold relativistic fluid equations
+     * using a Muscl-Handcock scheme
+     */
+    /**
+     * \brief Advective term, cold-rel. fluids
+     *
+     * \param[in] lev refinement level
+     */
+    void AdvectivePush_Muscl (int lev);
+
+
+    /**
+    * Apply (non-periodic) BC on the fluids (needed for spatial derivative),
+    * and communicate N, NU at boundaries
+    */
+    /**
+     * \brief Apply non-periodic BC to fluids and communicate boundaries
+     *
+     * \param[in] lev refinement level
+     */
+    void ApplyBcFluidsAndComms (int lev);
+
+    /**
+     * centrifugal_source adds contributions due to curvature acceleration for a
+     * single timestep using an SSP-RK3 timestep
+    */
+    /**
+     * \brief Centrifugal source term
+     *
+     * \param[in] lev refinement level
+     */
+    void centrifugal_source (int lev);
+
+    /**
+     * GatherAndPush introduces the lorentz term in the cold relativistic fluid
+     * equations for a single timestep (dt) using Higuera and Cary Push
+     */
+    /**
+     * \brief Lorentz Momentum Source
+     *
+     * \param[in] lev refinement level
+     * \param[in] Ex Yee electric field (x)
+     * \param[in] Ey Yee electric field (y)
+     * \param[in] Ez Yee electric field (z)
+     * \param[in] Bx Yee magnetic field (x)
+     * \param[in] By Yee magnetic field (y)
+     * \param[in] Bz Yee magnetic field (z)
+     * \param[in] cur_time Current time
+     */
+    void GatherAndPush (int lev,
+        const amrex::MultiFab& Ex, const amrex::MultiFab& Ey, const amrex::MultiFab& Ez,
+        const amrex::MultiFab& Bx, const amrex::MultiFab& By, const amrex::MultiFab& Bz,
+        amrex::Real cur_time);
+
+    /**
+     * DepositCurrent interpolates the fluid current density comps. onto the Yee grid and
+     * sums the contributions to the particle current density
+     */
+    /**
+     * \brief Deposit fluid current density.
+     *
+     * \param[in] lev refinement level
+     * \param[in,out] jx current density MultiFab x comp.
+     * \param[in,out] jy current density MultiFab y comp.
+     * \param[in,out] jz current density MultiFab z comp.
+     */
+    void DepositCurrent (int lev,
+        amrex::MultiFab& jx, amrex::MultiFab& jy, amrex::MultiFab& jz);
+
+    /**
+     * DepositCharge deposits density onto rho
```

Comment:
Maybe the docstring could be modified to be consistent with the docstring of `DepositCurrent`

Reply:
Done.